.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "Embedded-Base"]
 	path = Embedded-Base
 	url = ../Embedded-Base
```

Comment:
this needs to be changed to the URL of the github repo.  My b for missing that on prev PR.

Reply:
stale review dismissal is my arch nemesis